__tests__/integrations.test.js

```diff
@@ -169,6 +168,29 @@ describe('Articles', () => {
           expect(errorMessage).toBe(expectedResponse);
         });
     });
+    test('GET 200: Endpoint returns an a singular article by ID, with a comment count', () => {
+      return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body: { article } }) => {
+          const expectedArticle = {
+            article_id: 1,
```

Comment:
No need to test for all of the properties that we have previously tested for, we should just create a new test and use a flexible jest matcher like .toMatchObject to assert against the one new property here. Both tests should then pass and we haven't need to go back and edit old tests which is bad practice 👍